widgetssdk/build.gradle

```diff
@@ -55,7 +54,6 @@ dependencies {
     // https://square.github.io/leakcanary/changelog/
     // debugImplementation "com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion"
 
-    // Replaced with a local AAR file
```

Comment:
I think this line shouldn't be deleted

Reply:
to be honest, I don't understand why we need this comment, in fact, dependency is not replaced with local aar.

Reply:
we can add the local arr file and disable dependency manually, but this is only for test purposes on the local environment, that's why I found that line pointless.

Reply:
I see.

Maybe it makes sense to edit the comment?

I mean change "Replaced with a local AAR file" to something like "Disable this line if dependency shouldbe taken from local AAR file".

Although it's not related to the current feature. 🙂